evap/evaluation/templates/base.html

```diff
@@ -14,7 +14,9 @@
         <title>{% block title %}EvaP{% endblock %}</title>
 
         {% compress css %}
-            <link rel="stylesheet" href="{% get_static_prefix %}font-awesome/css/font-awesome.min.css" />
+            <link rel="stylesheet" href="{% get_static_prefix %}font-awesome/css/fontawesome.min.css" />
+            <link rel="stylesheet" href="{% get_static_prefix %}font-awesome/css/fa-solid.min.css" />
+            <link rel="stylesheet" href="{% get_static_prefix %}font-awesome/css/fa-regular.min.css" />
```

Comment:
if you could give a brief explanation or point to an explanation why these additional files are necessary and why each of them has 5 different versions, that would be nice.

Reply:
there is now only one version left (i wasn't quite done). after consulting https://www.w3schools.com/css/css3_fonts.asp, we're using WOFF.

what kind of explanation should i add? this is plain font awesome 5, it consists of several fonts internally - these are the ones we need for the icons we're using. no magic involved.

Reply:
firefox and chrome both will first request the woff2 font, and upon getting a 404, fall back to the woff font. i would suggest re-adding the woff2 versions - and maybe even removing the other fonts from the small css files for correctness? btw https://caniuse.com/#search=woff tells a better story for woff2 than the w3schools page.

Reply:
done